nessie/golden/lakectl_doctor_long_secret_access_key.golden

```diff
@@ -0,0 +1,6 @@
+secret_access_key value looks suspicious...
+It looks like you have a problem with your '.lakectl.yaml' file. 
+It is possible that the access_key_id' or 'secret_access_key' you supplied are wrong 
+
+
+Error executing command.
```

Comment:
This file is identical to `nessie/golden/lakectl_doctor_invalid_char_secret_access_key.golden`
You can (and should) use the same golden file, rather than have multiple identical files
I would go with some generaic name, like `lakectl_doctor_invalid_secret_access_key` and use it wherever it needed

Reply:
removed some of the tests and combined the golden files where it was possible